### ISSUE TITLE
Feature request - cmake-version: 'latest'

### ISSUE THREAD

**Issue by carlosperate** [jwlawson/actions-setup-cmake#41]
To be able to automatically fetch the latest version available.

This is useful with a job matrix to test that libraries always compile with the latest version of CMake without having to constantly update the workflow file:

```yaml
jobs:
  build-test:
    strategy:
      matrix:
        # Oldest supported and newest release
        cmake: ['3.4.0', 'latest']
    steps:
      - name: Setup CMake ${{ matrix.cmake }}
        uses: jwlawson/actions-setup-cmake@v1
        with:
          cmake-version: ${{ matrix.cmake }}
```

**Comment by jwlawson** [comment]
This is already possible by giving an empty string to `cmake-version` (which is also the default), similar to what we do [here](https://github.com/jwlawson/actions-setup-cmake/blob/master/.github/workflows/main.yml#L16).

I appreciate it's not very obvious so adding an explicit alias for 'latest' would definitely help make workflow files more readable.

**Comment by carlosperate** [comment]
Oh that's great! And after reading the README again I noticed it's already described there, so that was my bad for not spotting it on my first pass 😅
An alias would definitely help make it more obvious when somebody unfamiliar reads the workflows (otherwise I'd likely be adding a comment to clarify it), thanks!

**Comment by jwlawson** [comment]
This is now available in [version 1.12](https://github.com/jwlawson/actions-setup-cmake/releases/tag/v1.12). Thanks for the suggestion!

**Comment by carlosperate** [comment]
Awesome, thanks for the incredibly quick response! 🚀
